presto-main/src/main/java/io/prestosql/operator/HashSemiJoinOperator.java

```diff
@@ -71,18 +75,24 @@ public void noMoreOperators()
         @Override
         public OperatorFactory duplicate()
         {
-            return new HashSemiJoinOperatorFactory(operatorId, planNodeId, setSupplier, probeTypes, probeJoinChannel);
+            return new HashSemiJoinOperatorFactory(operatorId, planNodeId, setSupplier, probeTypes, probeJoinChannel, probeJoinHashChannel);
         }
     }
 
     private final int probeJoinChannel;
     private final ListenableFuture<ChannelSet> channelSetFuture;
+    private final Optional<Integer> probeHashChannel;
 
     private ChannelSet channelSet;
     private Page outputPage;
     private boolean finishing;
 
     public HashSemiJoinOperator(OperatorContext operatorContext, SetSupplier channelSetFuture, int probeJoinChannel)
```

Comment:
`HashSemiJoinOperator` is an internal detail of Presto, so we don't need to maintain API backwards compatibility.  This constructor is no longer used, so let's just remove it.